File: RST/app/src/main/java/com/unimaq/rst/entities/Atencion.kt
package com.unimaq.rst.entities

import java.util.Date

data class Atencion (
    var id:Long,
    val numero_solicitud:String,
    val id_maquinaria:Long,
    val numero_serie:String,
    val anho_fabricacion:Int,
    val horometro:String,
    val stir_2:String,
    val revision_hidraulica:Boolean,
    val revision_aceite:Boolean,
    val revision_calibracion:Boolean,
    val revision_neumaticos:Boolean,
    val observaciones:String,
    val firma_tecnico:String,
    val firma_supervisor:String,
    val fecha_creacion: String,
    val fecha_modificacion: String,
    val maquinaria_marca:String,
    val maquinaria_categoria:String,
    val maquinaria_modelo:String,
    val maquinaria_imagen:String,
    val fecha_creacion_formato:String
){
    fun maquinaria():String{
        return "$maquinaria_marca - $maquinaria_categoria $maquinaria_modelo"
    }

}

File: RSTLambda/RSTLambda.Entities/Atencion.cs
namespace RSTLambda.Entities
{
    public class Atencion
    {
        public long Id { get; set; }
        public string NumeroSolicitud { get; set; }
        public long IdMaquinaria { get; set; }
        public string NumeroSerie { get; set; }
        public int AnhoFabricacion { get; set; }
        public string UltimoMantenimiento { get; set; }
        public string Horometro { get; set; }
        public string Stir2 { get; set; }
        public bool RevisionHidraulica { get; set; }
        public bool RevisionAceite { get; set; }
        public bool RevisionCalibracion { get; set; }
        public bool RevisionNeumaticos { get; set; }
        public string Observaciones { get; set; }
        public string FirmaTecnico { get; set; }
        public string FirmaSupervisor { get; set; }
        public DateTime FechaCreacion { get; set; }
        public DateTime FechaModificacion { get; set; }
        public string MaquinariaMarca { get; set; }
        public string MaquinariaCategoria { get; set; }
        public string MaquinariaModelo { get; set; }
        public string MaquinariaImagen { get; set; }
    }
}
